app/models/quote.rb

```diff
@@ -0,0 +1,99 @@
+class Quote < ActiveRecord::Base
+
+  def self.get_rates(request)
+    parsed_request = JSON.parse(request)
+    @quote = self.new
+    @quote.request = request
+    @origin = Quote.get_origin
+    @packages = Quote.get_packages(parsed_request["number_of_items"])
```

Comment:
Thank you! That's something I need to work on, I make them instance variables just as a habit but I need to make sure to use local variables when appropriate